apps/src/lib/kits/maker/redux.js

```diff
@@ -89,8 +89,7 @@ export function reducer(state = new MakerState(), action) {
     return state.set('connectionState', CONNECTING);
   } else if (action.type === REPORT_CONNECTED) {
     return state.merge({
-      connectionState: CONNECTED,
-      usingFakeBoardNextTime: false
```

Comment:
It seems odd to me that we'd have a real board be connected and still have usingFakeBoardNextTime set to true. Is there somewhere we're calling reportConnected() when we shouldn't be?

Reply:
Addressed this in comment below.

Reply:
Ahh ok. Makes sense. Thanks for the clarification!